tests/services/test_load_db.py

```diff
@@ -0,0 +1,180 @@
+        assert len(s1.access_types) == 2
+        assert s1.access_types[0].name == data["access_types"][0]["name"]
+        assert s1.access_types[1].name == data["access_types"][1]["name"]
+        assert len(s1.access_modes) == 2
+        assert s1.access_modes[0].name == data["access_modes"][0]["name"]
+        assert s1.access_modes[1].name == data["access_modes"][1]["name"]
+        assert len(s1.life_cycle_statuses) == 2
+        assert s1.life_cycle_statuses[0].name == data["life_cycle_statuses"][0]["name"]
+        assert s1.life_cycle_statuses[1].name == data["life_cycle_statuses"][1]["name"]
+        assert len(s1.trls) == 2
+        assert s1.trls[0].name == data["trls"][0]["name"]
+        assert s1.trls[1].name == data["trls"][1]["name"]
+
+        assert s2.id == data["services"][1]["id"]
+        assert s2.name == data["services"][1]["name"]
+        assert s2.description == data["services"][1]["description"]
+        assert s2.tagline == data["services"][1]["tagline"]
+        assert s2.countries == data["services"][1]["countries"]
+        assert len(s2.categories) == 1
+        assert s2.categories[0].name == data["categories"][1]["name"]
+        assert len(s2.providers) == 1
+        assert s2.providers[0].name == data["providers"][1]["name"]
+        assert s2.resource_organisation.name == data["providers"][1]["name"]
+        assert len(s2.scientific_domains) == 1
+        assert s2.scientific_domains[0].name == data["scientific_domains"][1]["name"]
+        assert len(s2.platforms) == 1
+        assert s2.platforms[0].name == data["platforms"][1]["name"]
+        assert len(s2.target_users) == 1
+        assert s2.target_users[0].name == data["target_users"][1]["name"]
+        assert len(s2.related_services) == 1
+        assert s2.related_services[0] == data["services"][0]["id"]
+        assert len(s2.required_services) == 0
+        assert len(s2.access_types) == 1
+        assert s2.access_types[0].name == data["access_types"][1]["name"]
+        assert len(s2.access_modes) == 1
+        assert s2.access_modes[0].name == data["access_modes"][1]["name"]
+        assert len(s2.life_cycle_statuses) == 1
+        assert s2.life_cycle_statuses[0].name == data["life_cycle_statuses"][1]["name"]
+        assert len(s2.trls) == 1
+        assert s2.trls[0].name == data["trls"][1]["name"]
+
+        assert len(u.scientific_domains) == 2
+        assert u.scientific_domains[0].name == data["scientific_domains"][0]["name"]
+        assert u.scientific_domains[1].name == data["scientific_domains"][1]["name"]
+        assert len(u.categories) == 2
+        assert u.categories[0].name == data["categories"][0]["name"]
+        assert u.categories[1].name == data["categories"][1]["name"]
+        assert len(u.accessed_services) == 2
+        assert u.accessed_services[0].name == data["services"][0]["name"]
+        assert u.accessed_services[1].name == data["services"][1]["name"]
```

Comment:
It's not necessarily bad but I think it would be much cleaner to just compare two lists / dicts together - pytest is smart enough to make it work relatively painfree.